swaglyrics/__main__.py

```diff
@@ -25,10 +25,15 @@ def main():
 	print("Updated unsupported.txt successfully.")
 
 	parser = argparse.ArgumentParser(
+		prog="Swaglyrics",
+		usage='%(prog)s [options]',
```

Comment:
str.format() is a better way in my opinion 🙂

Reply:
I have no idea if we can use the parameter of an object in `.format()` . I did as per the docs of argparse, you can check in the docs --> [Argparse - usage](https://docs.python.org/3/library/argparse.html#usage)

Reply:
Cool enough

Reply:
Coverage also increased by +0.6%, I'm glad about that. Can it be merged now? 🤔

Reply:
I'll test it out in the evening then if everything is cool, we'll merge.

Reply:
Checked? Is there anything which isn't working? ☺️

Reply:
I've not been able to make out time. Don't worry about it 🙂 
You don't have to do anything here for now, you can focus on other stuff 👍

Reply:
Alright, I guess you're focussing on your exams, best of luck for your exams man 😄

Reply:
Yes, thank you 🙂